app/helpers/test_helper.rb

```diff
@@ -3,7 +3,8 @@
 
 module TestHelper
   # Consider this a fixture for tests. Needs to integrate with dag_json for public_sample_run_stage.
-  TEST_RESULT_FOLDER = %w[
+  # Avoid "warning: already initialized constant" by conditional assignment
```

Comment:
It looks like this was being cause by including `test_helper` more than once, but I'm not sure how to fix that